<!-- 
RSS generated by JIRA (6.3.4#6332-sha1:51bc225ef474afe3128b2f66878477f322397b16) at Sun May 17 03:09:17 UTC 2015

It is possible to restrict the fields that are returned in this document by specifying the 'field' parameter in your request.
For example, to request only the issue key and summary add field=key&field=summary to the URL of your request.
For example:
https://issues.apache.org/jira/si/jira.issueviews:issue-xml/DERBY-5456/DERBY-5456.xml?field=key&amp;field=summary
-->
<rss version="0.92" >
<channel>
    <title>ASF JIRA</title>
    <link>https://issues.apache.org/jira</link>
    <description>This file is an XML representation of an issue</description>
    <language>en-uk</language>    <build-info>
        <version>6.3.4</version>
        <build-number>6332</build-number>
        <build-date>15-08-2014</build-date>
    </build-info>

<item>
            <title>[DERBY-5456] Problems running SecureServerTest and replication tests on Open JDK 7 for Mac OS X</title>
                <link>https://issues.apache.org/jira/browse/DERBY-5456</link>
                <project id="10594" key="DERBY">Derby</project>
                    <description>&lt;p&gt;I see the following errors when I run the JUnit tests on Open JDK 7 for Mac OS X:&lt;/p&gt;

&lt;p&gt;sanity is sane&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........START-SPAWNED:sysinfo STANDARD OUTPUT: exit code=1&lt;br/&gt;
Tue Oct 11 10:15:02 PDT 2011 : Error getting SysInfo information.&lt;br/&gt;
END-SPAWNED  :sysinfo STANDARD OUTPUT:&lt;br/&gt;
FSTART-SPAWNED:SpawnedNetworkServer STANDARD OUTPUT: exit code=143&lt;br/&gt;
Tue Oct 11 10:14:54 PDT 2011 : Security manager installed using the Basic server security policy.&lt;br/&gt;
Tue Oct 11 10:15:01 PDT 2011 : Warning: UnknkownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known.&lt;br/&gt;
Tue Oct 11 10:15:01 PDT 2011 : Apache Derby Network Server - 10.8.2.2 - (1181258) started and ready to accept connections on port 1527&lt;br/&gt;
Tue Oct 11 10:15:02 PDT 2011 : host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
Tue Oct 11 10:15:02 PDT 2011 : host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer STANDARD OUTPUT:&lt;br/&gt;
E.START-SPAWNED:sysinfo STANDARD OUTPUT: exit code=1&lt;br/&gt;
Tue Oct 11 10:15:50 PDT 2011 : Error getting SysInfo information.&lt;br/&gt;
END-SPAWNED  :sysinfo STANDARD OUTPUT:&lt;br/&gt;
FSTART-SPAWNED:SpawnedNetworkServer STANDARD OUTPUT: exit code=143&lt;br/&gt;
Tue Oct 11 10:15:43 PDT 2011 : Security manager installed using the Basic server security policy.&lt;br/&gt;
Tue Oct 11 10:15:50 PDT 2011 : Warning: UnknkownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known.&lt;br/&gt;
Tue Oct 11 10:15:50 PDT 2011 : Apache Derby Network Server - 10.8.2.2 - (1181258) started and ready to accept connections on port 1527&lt;br/&gt;
Tue Oct 11 10:15:50 PDT 2011 : host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
Tue Oct 11 10:15:50 PDT 2011 : host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer STANDARD OUTPUT:&lt;br/&gt;
E...............................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
...................START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F.START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F.START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F....................&lt;br/&gt;
START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F.....START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F......................START-SPAWNED:SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;br/&gt;
END-SPAWNED  :SpawnedNetworkServer ERROR OUTPUT:&lt;br/&gt;
F..............&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
.........................................&lt;br/&gt;
....................E.E.E.F.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E&lt;br/&gt;
Time: 5,033.075&lt;br/&gt;
There were 20 errors:&lt;br/&gt;
1) SecureServerTest( Opened = false, Authenticated= true, CustomDerbyProperties= null, WildCardHost= 0.0.0.0 )java.lang.Exception: DRDA_NeedLocalHost.S:host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessageWork(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessage(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.readResult(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.shutdown(Unknown Source)&lt;br/&gt;
	at org.apache.derby.drda.NetworkServerControl.shutdown(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.junit.NetworkServerTestSetup.tearDown(NetworkServerTestSetup.java:379)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:22)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
2) SecureServerTest( Opened = false, Authenticated= true, CustomDerbyProperties= null, WildCardHost= 0.00.000.0 )java.lang.Exception: DRDA_NeedLocalHost.S:host: localhost is not local to the server running on 0.0.0.0, so cannot be used for NetworkServerControl commands  &lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessageWork(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessage(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.readResult(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.drda.NetworkServerControlImpl.shutdown(Unknown Source)&lt;br/&gt;
	at org.apache.derby.drda.NetworkServerControl.shutdown(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.junit.NetworkServerTestSetup.tearDown(NetworkServerTestSetup.java:379)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:22)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
3) testReplication_Local_LogFilesSynched(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local.testReplication_Local_LogFilesSynched(ReplicationRun_Local.java:195)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
4) testReplication_Local_1_InsertUpdateDeleteOnMaster(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_1)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_1.testReplication_Local_1_InsertUpdateDeleteOnMaster(ReplicationRun_Local_1.java:77)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
5) testReplication_Local_1_Indexing(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_1Indexing)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.makeReadyForReplication(ReplicationRun.java:2518)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_1Indexing.testReplication_Local_1_Indexing(ReplicationRun_Local_1Indexing.java:70)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
6) testReplication_Local_StateTest_part1_1(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_1)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_1.testReplication_Local_StateTest_part1_1(ReplicationRun_Local_StateTest_part1_1.java:96)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
7) testReplication_Local_StateTest_part1_2(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_2)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_2.testReplication_Local_StateTest_part1_2(ReplicationRun_Local_StateTest_part1_2.java:104)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
8) testReplication_Local_StateTest_part1_3(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_3)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1_3.testReplication_Local_StateTest_part1_3(ReplicationRun_Local_StateTest_part1_3.java:95)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
9) testReplication_Local_StateTest_part2(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part2)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part2.testReplication_Local_StateTest_part2(ReplicationRun_Local_StateTest_part2.java:97)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
10) testReplication_Local_3_p1_StateNegativeTests(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p1)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p1.testReplication_Local_3_p1_StateNegativeTests(ReplicationRun_Local_3_p1.java:98)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
11) testReplication_Local_3_p2_StateTests_bigInsert_sleepBeforeStopMaster(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.replication_Local_3_p2_StateTests(ReplicationRun_Local_3_p2.java:103)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.testReplication_Local_3_p2_StateTests_bigInsert_sleepBeforeStopMaster(ReplicationRun_Local_3_p2.java:248)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
12) testReplication_Local_3_p2_StateTests_smallInsert_sleepBeforeStopMaster(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.replication_Local_3_p2_StateTests(ReplicationRun_Local_3_p2.java:103)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.testReplication_Local_3_p2_StateTests_smallInsert_sleepBeforeStopMaster(ReplicationRun_Local_3_p2.java:253)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
13) testReplication_Local_3_p2_StateTests_bigInsert_immediateStopMaster(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.replication_Local_3_p2_StateTests(ReplicationRun_Local_3_p2.java:103)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p2.testReplication_Local_3_p2_StateTests_bigInsert_immediateStopMaster(ReplicationRun_Local_3_p2.java:233)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
14) testReplication_Local_3_p3_StateNegativeTests(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p3)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p3.testReplication_Local_3_p3_StateNegativeTests(ReplicationRun_Local_3_p3.java:103)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
15) testReplication_Local_3_p4_StateNegativeTests(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p4)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p4.testReplication_Local_3_p4_StateNegativeTests(ReplicationRun_Local_3_p4.java:100)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
16) testReplication_Local_3_p5_DERBY_3878(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p5)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.makeReadyForReplication(ReplicationRun.java:2518)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p5.testReplication_Local_3_p5_DERBY_3878(ReplicationRun_Local_3_p5.java:73)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
17) testReplication_Encrypted_1_miniLoad_negative(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Encrypted_1)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.makeReadyForReplication(ReplicationRun.java:2518)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Encrypted_1.testReplication_Encrypted_1_miniLoad_negative(ReplicationRun_Local_Encrypted_1.java:132)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
18) testReplication_Encrypted_1_stdLoad(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Encrypted_1)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.makeReadyForReplication(ReplicationRun.java:2518)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Encrypted_1.testReplication_Encrypted_1_stdLoad(ReplicationRun_Local_Encrypted_1.java:77)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
19) testReplication_Local_3_p6_autocommit_OK(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p6)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XBCA0, SQLERRMC: ContainerCacheContainer(0, 16)XBCA0.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.PreparedStatement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p6.derby_3896(ReplicationRun_Local_3_p6.java:144)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_3_p6.testReplication_Local_3_p6_autocommit_OK(ReplicationRun_Local_3_p6.java:86)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XBCA0, SQLERRMC: ContainerCacheContainer(0, 16)XBCA0.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parseEXCSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetPreparedStatement.readExecute_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.PreparedStatement.readExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.PreparedStatement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.PreparedStatement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
20) testSlaveWaitsForMaster(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Derby4910)java.sql.SQLException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.execute(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.bootMasterDatabase(ReplicationRun.java:868)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_Derby4910.testSlaveWaitsForMaster(ReplicationRun_Local_Derby4910.java:80)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: org.apache.derby.client.am.SqlException: DERBY SQL error: SQLCODE: -1, SQLSTATE: XSAI2, SQLERRMC: 384XSAI2.S&lt;br/&gt;
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePrepareError(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.StatementReply.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.net.NetStatement.readPrepare_(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.readPrepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)&lt;br/&gt;
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)&lt;br/&gt;
	... 32 more&lt;br/&gt;
There were 9 failures:&lt;br/&gt;
1) SecureServerTest( Opened = false, Authenticated= true, CustomDerbyProperties= null, WildCardHost= 0.0.0.0 )junit.framework.AssertionFailedError: expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.runServerCommand(SecureServerTest.java:501)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.runsysinfo(SecureServerTest.java:415)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.testServerStartup(SecureServerTest.java:356)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
2) SecureServerTest( Opened = false, Authenticated= true, CustomDerbyProperties= null, WildCardHost= 0.00.000.0 )junit.framework.AssertionFailedError: expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.runServerCommand(SecureServerTest.java:501)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.runsysinfo(SecureServerTest.java:415)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.derbynet.SecureServerTest.testServerStartup(SecureServerTest.java:356)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
3) JMXTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
4) ManagementMBeanTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
5) InactiveManagementMBeanTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
6) VersionMBeanTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
7) JDBCMBeanTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
8) NetworkServerMBeanTest:clientjunit.framework.AssertionFailedError: Timed out waiting for network server to start:Spawned SpawnedNetworkServer exitCode=1&lt;br/&gt;
STDERR:&lt;br/&gt;
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known&lt;/p&gt;

&lt;p&gt;	at org.apache.derbyTesting.junit.NetworkServerTestSetup.setUp(NetworkServerTestSetup.java:205)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:20)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
9) testReplication_Local_StateTest_part1(org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1)junit.framework.AssertionFailedError: Unexpected SQLException: -1 08004 DERBY SQL error: SQLCODE: -1, SQLSTATE: 08004, SQLERRMC: Connection refused : java.sql.SQLException: Required property &apos;slaveHost&apos; has not been set.&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1._testPreStartedMasterServer(ReplicationRun_Local_StateTest_part1.java:212)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun_Local_StateTest_part1.testReplication_Local_StateTest_part1(ReplicationRun_Local_StateTest_part1.java:88)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.replicationTests.ReplicationRun.runBare(ReplicationRun.java:206)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;/p&gt;

&lt;p&gt;FAILURES!!!&lt;br/&gt;
Tests run: 11559,  Failures: 9,  Errors: 20&lt;/p&gt;</description>
                <environment>mac os x 10.6.8, OpenJDK Runtime Environment (build 1.7.0-b210-20110929)</environment>
        <key id="12526723">DERBY-5456</key>
            <summary>Problems running SecureServerTest and replication tests on Open JDK 7 for Mac OS X</summary>
                <type id="1" iconUrl="https://issues.apache.org/jira/images/icons/issuetypes/bug.png">Bug</type>
                                            <priority id="3" iconUrl="https://issues.apache.org/jira/images/icons/priorities/major.png">Major</priority>
                        <status id="6" iconUrl="https://issues.apache.org/jira/images/icons/statuses/closed.png" description="The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.">Closed</status>
                    <statusCategory id="3" key="done" colorName="green"/>
                                    <resolution id="1">Fixed</resolution>
                                        <assignee username="-1">Unassigned</assignee>
                                    <reporter username="rhillegas">Rick Hillegas</reporter>
                        <labels>
                    </labels>
                <created>Tue, 11 Oct 2011 20:14:03 +0100</created>
                <updated>Thu, 1 Nov 2012 21:10:39 +0000</updated>
                            <resolved>Mon, 17 Oct 2011 17:25:19 +0100</resolved>
                                    <version>10.8.2.2</version>
                                    <fixVersion>10.8.3.0</fixVersion>
                    <fixVersion>10.9.1.0</fixVersion>
                                    <component>Test</component>
                        <due></due>
                            <votes>0</votes>
                                    <watches>0</watches>
                                                                <comments>
                            <comment id="13125344" author="kristwaa" created="Tue, 11 Oct 2011 21:10:41 +0100"  >&lt;p&gt;This may be a machine configuration issue. Some questions:&lt;br/&gt;
Do you have an entry for richard-hillegas-computer.local (or maybe without .local will suffice) in /etc/hosts, or can it otherwise be resolved?&lt;br/&gt;
Is there more than one NIC on the computer?&lt;br/&gt;
Are you running both IPv4 and IPv6?&lt;/p&gt;

&lt;p&gt;Another issue would be to figure out how these tests differ from the other network tests spawning a network server when it comes to obtaining/resolving the host name.&lt;/p&gt;</comment>
                            <comment id="13125835" author="rhillegas" created="Wed, 12 Oct 2011 14:20:02 +0100"  >&lt;p&gt;Thanks for those questions, Kristian. Here are some answers:&lt;/p&gt;

&lt;p&gt;1) Do you have an entry for richard-hillegas-computer.local (or maybe without .local will suffice) in /etc/hosts, or can it otherwise be resolved?&lt;/p&gt;

&lt;p&gt;Nope. /etc/hosts has entries for localhost and broadcasthost but not for richard-hillegas-computer.local. That latter, unknown name is what is printed out when I type the hostname command.&lt;/p&gt;

&lt;p&gt;2) Is there more than one NIC on the computer?&lt;/p&gt;

&lt;p&gt;Not that I can see.&lt;/p&gt;

&lt;p&gt;3) Are you running both IPv4 and IPv6? &lt;/p&gt;

&lt;p&gt;Yes.&lt;/p&gt;

&lt;p&gt;Thanks.&lt;/p&gt;</comment>
                            <comment id="13127676" author="rhillegas" created="Fri, 14 Oct 2011 17:34:40 +0100"  >&lt;p&gt;I have found a bug in the open jdk 7 preview for mac os x which causes server startup to cough up the following warning:&lt;/p&gt;

&lt;p&gt;Warning: UnknkownHostException: richard-hillegas-computer.local: richard-hillegas-computer.local: nodename nor servname provided, or not known. &lt;/p&gt;

&lt;p&gt;This bug is tracked by &lt;a href=&quot;http://java.net/jira/browse/MACOSX_PORT-564&quot; class=&quot;external-link&quot; rel=&quot;nofollow&quot;&gt;http://java.net/jira/browse/MACOSX_PORT-564&lt;/a&gt;&lt;/p&gt;

&lt;p&gt;In addition, it seems to me that server startup could be made more robust. Right now, it gives up collecting local addresses if it encounters a problem. If the server didn&apos;t give up, then it would eventually collect the local address it needs. I will attach a patch to make server startup more robust.&lt;/p&gt;</comment>
                            <comment id="13127683" author="rhillegas" created="Fri, 14 Oct 2011 17:40:30 +0100"  >&lt;p&gt;Attaching derby-5456-01-aa-moreRobustStartup.diff. This patch makes the server continue collecting local addresses if one of its attempts fails. The failures are not supposed to be fatal, they just result in the server printing a warning to derby.log. With this patch, SecureServerTest runs cleanly on the open jdk 7 preview for mac os x. I will run full regression tests now.&lt;/p&gt;

&lt;p&gt;Touches the following file:&lt;/p&gt;

&lt;p&gt;M       java/drda/org/apache/derby/impl/drda/NetworkServerControlImpl.java&lt;/p&gt;</comment>
                            <comment id="13127793" author="rhillegas" created="Fri, 14 Oct 2011 20:18:17 +0100"  >&lt;p&gt;Tests passed cleanly for me on Java 6 except for the error below. This error does not seem to me to be related to this change. Committed at subversion revision 1183463.&lt;/p&gt;

&lt;p&gt;Time: 5,975.721&lt;br/&gt;
There was 1 failure:&lt;br/&gt;
1) testDerby3980_repeatable_read(org.apache.derbyTesting.functionTests.tests.lang.DeadlockDetectionTest)junit.framework.ComparisonFailure: Unexpected SQL state. expected:&amp;lt;40&lt;span class=&quot;error&quot;&gt;&amp;#91;00&amp;#93;&lt;/span&gt;1&amp;gt; but was:&amp;lt;40&lt;span class=&quot;error&quot;&gt;&amp;#91;XL&amp;#93;&lt;/span&gt;1&amp;gt;&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.assertSQLState(BaseJDBCTestCase.java:791)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.assertSQLState(BaseJDBCTestCase.java:840)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.DeadlockDetectionTest.testDerby3980_repeatable_read(DeadlockDetectionTest.java:137)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&lt;br/&gt;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)&lt;br/&gt;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:113)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
Caused by: java.sql.SQLTransactionRollbackException: A lock could not be obtained within the time requested&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(SQLExceptionFactory40.java:89)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:256)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:400)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:348)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2290)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(ConnectionChild.java:82)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedStatement.executeStatement(EmbedStatement.java:1334)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedPreparedStatement.executeStatement(EmbedPreparedStatement.java:1692)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedPreparedStatement.executeUpdate(EmbedPreparedStatement.java:309)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.assertUpdateCount(BaseJDBCTestCase.java:1237)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.DeadlockDetectionTest$1.run(DeadlockDetectionTest.java:112)&lt;br/&gt;
Caused by: java.sql.SQLException: A lock could not be obtained within the time requested&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:45)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory40.java:122)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(SQLExceptionFactory40.java:71)&lt;br/&gt;
	... 10 more&lt;br/&gt;
Caused by: ERROR 40XL1: A lock could not be obtained within the time requested&lt;br/&gt;
	at org.apache.derby.iapi.error.StandardException.newException(StandardException.java:268)&lt;br/&gt;
	at org.apache.derby.impl.services.locks.ConcurrentLockSet.lockObject(ConcurrentLockSet.java:618)&lt;br/&gt;
	at org.apache.derby.impl.services.locks.AbstractPool.lockObject(AbstractPool.java:119)&lt;br/&gt;
	at org.apache.derby.impl.services.locks.ConcurrentPool.lockObject(ConcurrentPool.java:28)&lt;br/&gt;
	at org.apache.derby.impl.store.raw.xact.RowLocking3.lockRecordForRead(RowLocking3.java:187)&lt;br/&gt;
	at org.apache.derby.impl.store.access.conglomerate.OpenConglomerate.lockPositionForRead(OpenConglomerate.java:466)&lt;br/&gt;
	at org.apache.derby.impl.store.access.conglomerate.GenericScanController.fetchRows(GenericScanController.java:683)&lt;br/&gt;
	at org.apache.derby.impl.store.access.heap.HeapScan.fetchNext(HeapScan.java:238)&lt;br/&gt;
	at org.apache.derby.impl.sql.execute.TableScanResultSet.getNextRowCore(TableScanResultSet.java:577)&lt;br/&gt;
	at org.apache.derby.impl.sql.execute.ProjectRestrictResultSet.getNextRowCore(ProjectRestrictResultSet.java:261)&lt;br/&gt;
	at org.apache.derby.impl.sql.execute.DMLWriteResultSet.getNextRowCore(DMLWriteResultSet.java:127)&lt;br/&gt;
	at org.apache.derby.impl.sql.execute.UpdateResultSet.collectAffectedRows(UpdateResultSet.java:440)&lt;br/&gt;
	at org.apache.derby.impl.sql.execute.UpdateResultSet.open(UpdateResultSet.java:264)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericPreparedStatement.executeStmt(GenericPreparedStatement.java:478)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericPreparedStatement.execute(GenericPreparedStatement.java:319)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedStatement.executeStatement(EmbedStatement.java:1242)&lt;br/&gt;
	... 4 more&lt;/p&gt;

&lt;p&gt;FAILURES!!!&lt;br/&gt;
Tests run: 11647,  Failures: 1,  Errors: 0&lt;/p&gt;</comment>
                            <comment id="13127865" author="knutanders" created="Fri, 14 Oct 2011 22:13:45 +0100"  >&lt;p&gt;I suppose it would have been more robust to use InetAddress.getLoopbackAddress(), since it doesn&apos;t depend on name resolution. Unfortunately, that method is only available on Java 7.&lt;/p&gt;</comment>
                            <comment id="13128970" author="rhillegas" created="Mon, 17 Oct 2011 17:16:42 +0100"  >&lt;p&gt;All tests passed cleanly for my on the open jdk 7 preview for mac os x. That includes SecureServerTest and the replication tests.&lt;/p&gt;</comment>
                            <comment id="13128976" author="rhillegas" created="Mon, 17 Oct 2011 17:24:03 +0100"  >&lt;p&gt;Ported 1183463 from trunk to 10.8 branch at subversion revision 1185277.&lt;/p&gt;</comment>
                            <comment id="13164535" author="rhillegas" created="Wed, 7 Dec 2011 17:18:35 +0000"  >&lt;p&gt;I have verified that these errors do not occur any more using the latest build (220) of the OpenJDK 7 preview for Mac OS X. I verified this by running the full regression suite on build 220 using Derby 10.8.2.2. I did see 7 unrelated errors (the trigger-related errors in the upgrade tests, which have been fixed on the trunk).&lt;/p&gt;</comment>
                    </comments>
                    <attachments>
                            <attachment id="12499061" name="derby-5456-01-aa-moreRobustStartup.diff" size="1625" author="rhillegas" created="Fri, 14 Oct 2011 17:40:30 +0100"/>
                    </attachments>
                <subtasks>
                    </subtasks>
                <customfields>
                                                <customfield id="customfield_12310310" key="com.atlassian.jira.toolkit:attachments">
                        <customfieldname>Attachment count</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>1.0</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        <customfield id="customfield_12310220" key="com.atlassian.jira.ext.charting:firstresponsedate">
                        <customfieldname>Date of First Response</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>Tue, 11 Oct 2011 20:10:41 +0000</customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                                                                                                                                                                    <customfield id="customfield_12310420" key="com.pyxis.greenhopper.jira:gh-global-rank">
                        <customfieldname>Global Rank</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>59372</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                <customfield id="customfield_12311820" key="com.pyxis.greenhopper.jira:gh-lexo-rank">
                        <customfieldname>Rank</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>2|hy0efz:</customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                <customfield id="customfield_12310920" key="com.pyxis.greenhopper.jira:gh-global-rank">
                        <customfieldname>Rank (Obsolete)</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>36158</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                        <customfield id="customfield_12310222" key="com.atlassian.jira.ext.charting:timeinstatus">
                        <customfieldname>Time in Status</customfieldname>
                        <customfieldvalues>
                            
                        </customfieldvalues>
                    </customfield>
                                                                                            </customfields>
    </item>
</channel>
</rss>